### PR TITLE
Feature/format distance dynamically

### LINKED ISSUE #162: Display distances less than 1km in a nicer way
**Describe the bug**
Distances less than 1km are shown as 0km

**To Reproduce**

1. Go to website
2. Click on a date
3. Look for "Caring Clinic Doctors"
4. It's "0km away". It's about 100m away from city centre or something

**Screenshots**
image

**Additional context**
the km part is in the translation so I don't know if we should change it but 0km looks a bit odd.
should we use some distance formatting library which has i18n for this?

### PR DESCRIPTION
**Pull request by noway**
Resolves #162

In terms of timing, probably better to merge after Te Reo Maori and Hindi are done, so i can bring my branch up to date.

Can I use:
image

### CODE REVIEW

File: src/translations/locales/common_hi-HI.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "loading": "लोड हो रहा है",
     "getDirections": "दिशा - निर्देश प्राप्त करें",
     "makeABooking": "बुकिंग करें",
-    "kmAway": "{{distance}}किमी दूर",
```
Comment (noway) [inline src/translations/locales/common_hi-HI.json:10]: I tried my best here by matching the output. I'm 90% confident it's correct.

I believe कि means kilo and मी means meters.

File: src/translations/locales/common_zh-CN.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "loading": "正在加载...",
     "getDirections": "查询路线",
     "makeABooking": "预约",
-    "kmAway": "{{distance}} 公里",
```
Comment (noway) [inline src/translations/locales/common_zh-CN.json:10]: I tried my best here by matching the output. I'm 95% confident it's correct.

File: src/translations/locales/common_zh-TW.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "loading": "載入中...",
     "getDirections": "查詢路線",
     "makeABooking": "預約",
-    "kmAway": "{{distance}} 公里",
```
Comment (noway) [inline src/translations/locales/common_zh-TW.json:10]: I tried my best here by matching the output. I'm 95% confident it's correct.

**Review by noway — COMMENTED** [review]
Notes on translation changes I had to do.

**Review by wseagar — APPROVED** [review]
lgtm

**Pull request merged**